crates/core/src/kernel/models/actions.rs

```diff
@@ -268,6 +268,7 @@ impl Default for ProtocolInner {
 
 impl ProtocolInner {
     /// Create a new protocol action
+    #[allow(dead_code)]
```

Comment:
Crate: crates/core File: crates/core/src/kernel/models/actions.rs Clippy warnings fixed: "associated function new is never used --> crates/core/src/kernel/models/actions.rs:271:19" 

Not removed because this could break the API.